_includes/operations-reference.md

```diff
@@ -83,7 +83,7 @@ for the given operation.
 | :--------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Sorry, I just discovered that `operations-reference.md` is for Payment Orders, not for Payments. If we want to reuse the same include for all operations, we need to parametrize it so it changes operations and their description depending on which resource (`paymentorder`, `creditcard`, `invoice`, etc.) we are dealing with.

Reply:
Please elaborate on this ?

Reply:
As you removed `{% include operations-reference.md %}` in 1ccf65a02ec20feae8db2d4fdd2bfd02ec42cd14, I guess my comment is irrelevant, but that commit then begs the question: Don't we want a list of available operations for a card payment?

Reply:
ok, i understand. This i can do in a new branch. But do you want me to revert the changes back to the original in the meanwhile (in this branch )?

Reply:
No, all the changes are OK. Can you create a new task in Jira that describes this?

Reply:
Yes